### ISSUE TITLE
Why using X_test and y_test to train the model

### ISSUE THREAD

**Issue by yahayakayode** [beingaryan/Lung-Pneumonia-Prediction#1]
Why do you used X_test and y_test to validate the model as showing in the image below
history = model.fit(X_train, y_train, epochs=12, validation_data=(X_test, y_test), shuffle=True)
Aren’t you supposed to use the X_val and y_val?
